src/codeAnalysis/GlobalSuppressions.cs

```diff
@@ -45,3 +45,6 @@
 [assembly: SuppressMessage("Microsoft.Globalization", "CA1303:Do not pass literals as localized parameters", MessageId = "Microsoft.Templates.Core.Locations.JunctionNativeMethods.ThrowLastWin32Error(System.String)", Scope = "member", Target = "Microsoft.Templates.Core.Locations.JunctionNativeMethods.#InternalGetTarget(Microsoft.Win32.SafeHandles.SafeFileHandle)", Justification = "Only used for local generation")]
 [assembly: SuppressMessage("Microsoft.Globalization", "CA1303:Do not pass literals as localized parameters", MessageId = "Microsoft.Templates.Core.Locations.JunctionNativeMethods.ThrowLastWin32Error(System.String)", Scope = "member", Target = "Microsoft.Templates.Core.Locations.JunctionNativeMethods.#OpenReparsePoint(System.String,Microsoft.Templates.Core.Locations.JunctionNativeMethods+EFileAccess)", Justification = "Only used for local generation")]
 [assembly: SuppressMessage("Microsoft.Globalization", "CA1303:Do not pass literals as localized parameters", MessageId = "System.Windows.Documents.InlineCollection.Add(System.String)", Scope = "member", Target = "Microsoft.Templates.UI.Extensions.TextBlockExtensions.#OnSequentialFlowStepChanged(System.Windows.DependencyObject,System.Windows.DependencyPropertyChangedEventArgs)", Justification = "No text here")]
+
+// FxCop warning suppression for uninstantiated TestFixture classes
+[assembly: SuppressMessage("Microsoft.Performance", "CA1812: Avoid uninstantiated internal classes", Scope = "module", Justification = "The classes are used as TestFixture")]
```

Comment:
I don't understand why we would need this.  Can you point me to the offending line of code?

Reply:
This change was made to address @somil55 's comments above to add the suppression globally for better code readability,, referring to https://github.com/microsoft/PowerToys/pull/7312/commits/685e9034f3a022a5a075e1f12117a5a567eb1aa3.

Reply:
@ryanbodrug-microsoft CA1812 will be thrown for every file in the test project. This is mentioned here: https://github.com/dotnet/roslyn-analyzers/issues/1830. So rather than handling it manually in each file, it might be better to handle it in a global suppression file. The classes instantiated in this project are only for test purposes, so this should not affect anything else.

Reply:
Sounds good thanks for the info.  Can we update theJustification parameter to include the explanation from divyansh "CA1812 will be thrown for every file in the test project. This is mentioned here: dotnet/roslyn-analyzers#1830"